### PR TITLE
Initial Gravity Example dir and robot-shop-app cluster image

### PR DESCRIPTION
**Pull request by stevenGravy**
## Description
Initial examples directory and inclusion of robot-shop online web app example.  A tensor and other examples will be added shortly as well.

## Type of change
Document and example

## TODOs

- [X ] Perform manual testing
- [X] Write documentation
- [x] Address review feedback

## Testing done
Build and installation to a gravity 7.0 cluster

### CODE REVIEW

File: examples/robot-shop-app/resources/charts/robot-shop/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```shell
+$ helm install \
+    --set eum.key=xxxxxxxxx \
+    --set eum.url=https://eum-eu-west-1.instana.io \
+    ...
+```
+
+## Use with Minis
+
+When running on `minishift` or `minikube` set `nodeport` to true. The store will then be available on the IP address of your mini and node port of the web service.
```
Comment (a-palchikov) [inline examples/robot-shop-app/resources/charts/robot-shop/README.md:65]: nit: When running on `minishift` or `minikube`, set

File: examples/robot-shop-app/resources/charts/robot-shop/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+$ mini[kube|shift] ip
+192.168.66.101
+$ kubectl get svc web
+```
+
+Combine the IP and port number to make the URL `http://192.168.66.101:32145`
+
+### MiniShift
+
+Openshift is like K8s but not K8s. Set `openshift` to true or things will break. See the notes and scripts in the OpenShift directory of this repo.
```
Comment (a-palchikov) [inline examples/robot-shop-app/resources/charts/robot-shop/README.md:77]: Openshift is a Kubernetes distribution so I'm a bit puzzled what this wants to relay. Can you elaborate?
Maybe something along the line of:
```
If you want to test this on openshift with minishift, make sure to specify openshift=true to helm install:
```

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Examples
+
+Example cluster images for Gravity
```
Comment (r0mant) [inline examples/README.md:3]: Nit: End with a dot?

File: examples/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+
+- [Robot Shop](./robot-shop-app/README.md): End to end robot web online shop using microservices and multiple db backends.
```
Comment (r0mant) [inline examples/README.md:6]: Nit: Isn't it end-to-end (with dashes)?

File: examples/robot-shop-app/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+# Introduction
+
+This is an example online shopping app utilizing microservices in K8s with minor modifications from https://github.com/instana/robot-shop.  The modifications were to set registry helm varaibles and storage.  Note this image by default uses OpenEBS from gravity 7.0+.  On deployment you will get this front page.
```
Comment (r0mant) [inline examples/robot-shop-app/README.md:3]: Nit: Can we consistently capitalize **G**ravity everywhere it's mentioned?

Comment (r0mant) [inline examples/robot-shop-app/README.md:3]: varaibles -> variables

File: examples/robot-shop-app/README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+![Robot Shop](robotshop.png)
+
+
+## Building Image
+tele build -o robotshop.tar robot-shop-app/resources/app.yaml
+
+## Building App
```
Comment (r0mant) [inline examples/robot-shop-app/README.md:11]: Building Application Image. Also, should we briefly explain what "building cluster image" versus "building app image" does?

File: examples/robot-shop-app/README.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
+# Introduction
+
+This is an example online shopping app utilizing microservices in K8s with minor modifications from https://github.com/instana/robot-shop.  The modifications were to set registry helm varaibles and storage.  Note this image by default uses OpenEBS from gravity 7.0+.  On deployment you will get this front page.
+
+![Robot Shop](robotshop.png)
+
+
+## Building Image
```
Comment (r0mant) [inline examples/robot-shop-app/README.md:8]: Building Cluster Image

**Review by r0mant — APPROVED** [review]
Just a few typos/nits/suggestions.

File: examples/robot-shop-app/resources/charts/robot-shop/README.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+## Images
+
+By default the images are pulled from Docker Hub. Setting `image.repo` this can be changed, for example:
+
+```shell
+$ helm install --set image.repo=eu.gcr.io/acme ...
+```
+
+Will pull images from the European Google registry project `acme`.
```
[the remark this answers is not in the document]
Comment (stevenGravy) [reply examples/robot-shop-app/resources/charts/robot-shop/README.md]: this section was removed.

File: examples/robot-shop-app/resources/charts/robot-shop/README.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+By default the latest version of the images is pulled. A specific version can be used:
+
+```shell
+$ helm install --set image.version=0.1.2 ...
+```
+
+It is recommened to always use the latest version.
```
[the remark this answers is not in the document]
Comment (stevenGravy) [reply examples/robot-shop-app/resources/charts/robot-shop/README.md]: section was removed

**Pull request merged**